server/controllers/authentication/authentication.js

```diff
@@ -1,3 +1,13 @@
+const { verify } = require('jsonwebtoken');
+
 module.exports = (req, res, next) => {
-  res.send('Auth');
+  const { jwt } = req.cookies;
+  const secret = process.env.SECRET;
+  if (jwt && secret) {
+    verify(jwt, secret, (err, decoded) => {
+      if (decoded) req.auth = decoded;
```

Comment:
```suggestion
      if (decoded) req.user = decoded;
```
req.user does make more sense?